tests/main/prepare-image-classic/task.yaml

```diff
@@ -0,0 +1,75 @@
+summary: Check that prepare-image can prepare UC20+ classic models
+
+# running this on one machine is enough
+systems: [ubuntu-22.04-64]
+
+environment:
+    ROOT: /home/test/tmp/
+    STORE_ADDR: localhost:11028
+    STORE_DIR: $(pwd)/fake-store-blobdir
+
+prepare: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+
+    #shellcheck source=tests/lib/store.sh
+    . "$TESTSLIB"/store.sh
+    setup_fake_store "$STORE_DIR"
+
+    mkdir "$ROOT"
+    chown -R test:test "$ROOT"
+
+    # sign the model assertion
+    gendeveloper1 sign-model < "$TESTSLIB"/assertions/developer1-pi-20.model.json > "$ROOT/model.assertion"
+
+restore: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+
+    #shellcheck source=tests/lib/store.sh
+    . "$TESTSLIB"/store.sh
+    teardown_fake_store "$STORE_DIR"
+    rm -rf "$ROOT"
+
+execute: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+    if [ "$REMOTE_STORE" = "staging" ]; then
+        echo "SKIP: cannot run with staging store yet"
+        exit 0
+    fi
+
+    # get the model assertion
+    cp "$TESTSLIB"/assertions/developer1-22-classic-dangerous.model "$ROOT/model.assertion"
```

Comment:
I noticed we have `developer1-22-classic-dangerous.{json,model}`, shouldn't we have just the json and sign it as part of the test? (I guess the signing of the pi model was a typo?)

Reply:
Yes, fixed now.